m03_acos/create_acos_exercises.rst

```diff
@@ -0,0 +1,454 @@
+          }
+        }
+      },
+      "droppables":{
+        "source":{
+          "correct":[
+            "source"
+          ]
+        },
+        "date_folders":{
+          "correct":[
+            "date_folder"
+          ]
+        },
+        "experiments":{
+          "correct":[
+            "experiment"
+          ]
+        },
+        "files":{
+          "correct":[
+            "file"
+          ]
+        },
+        "readme":{
+          "correct":[
+            "readme"
+          ]
+        }
+      }
+    }
+  
+Once your Acos exercise is ready, and the exercise files have been created, you will have two choices for testing your code locally in the browser. :ref:`The first option <local-test>` is to test the exercise with a copy of the Acos-server in your computer. :ref:`The second option <docker-test>` is to test the exercise by using a pre-installed docker container which is downloaded after you have compiled an A+ course for the first time. 
+
+.. warning:: Before proceeding to test your Acos exercise, you should :ref:`configure the localhost <acos-docker-mac-windows>` in your computer. Therefore, you should verify that your host file have the ``127.0.0.1 acos`` configuration.
+
+
+.. _local-test:
+Test Acos activities locally
+------------------------------
+In order to test Acos exercises locally, you should download the `acos-server repository <https://github.com/acos-server/acos-server>`_ from GitHub. Once you have downloaded this file, you should link the :ref:`Acos exercise you have developed <create-acos-exercise>` to the local Acos-server.
+
+1. Open the terminal in the exercise project directory that you created :ref:`above <create-acos-exercise>`, and type:
+
+.. code-block:: bash
+
+  sudo npm link
+
+2. Open a terminal in the Acos server project directory and type: 
+  
+.. code-block:: bash
+
+  sudo npm link acos-draganddrop-data-management
+
+(the ``acos-draganddrop-data-management`` name will vary according to the name of your package).
+
+3. Now, it is time to run Acos server locally. For that, you need to open the terminal again in the Acos server project directory and type:
+
+.. code-block:: terminal
+  
+  node ./bin/www
+
+you can also type:
+
+.. code-block:: terminal
+  
+  npm start
+
+4. You can now open a web explorer and visit http://acos:3000. Your exercise should be there, under the draganddrop-data-management package.
+
+.. figure:: /images/gifs/AcosServerLocal.gif
+  :width: 100 %
+  :align: center
+  
+.. warning::
+  * Once you have create the` npm link <https://docs.npmjs.com/cli/link.html>`_ to the folder by typing following the steps 1-2 mentioned above, all the changes made in you exercises will be reflected in the ``node_modules`` folder of the Acos-server
+  * Normally, you will have to stop/start the node module every time you make changes in your exercise. (:kbd:`Ctrl` + :kbd:`c` to stop the module and ``npm start`` in the terminal to start the module).
```

Comment:
Is this a warning or kind of like a "notice this"? A warning gives at least me the feeling that "Beware! You might do something wrong, don't! Don't break anything!" Yet this box doesn't seem like it has any of that type of info.

Reply:
Should I use the important admonition instead?

Reply:
That could be good. `important` or `note` admonition could be used, whatever you think suits this best. Also, there seems to be a typo on line 371: "create" -> "created"